resources/packaging/archlinux/PKGBUILD.in

```diff
@@ -76,6 +76,16 @@ readonly _unwanted_bundled_libs=(
 depends+=(${_system_libs[@]} freetype2 harfbuzz)
 
 prepare() {
+  cd "$srcdir/ungoogled-chromium"
+
+  git checkout develop
```

Comment:
I'm a bit worried about this, because `develop` is a changing state. When we generate a PKGBUILD using buildkit, I'm thinking that we should substitute the current commit here (like it was done for `patch_hashes`) so we always checkout a consistent state. This makes it easier for reproducibility, and it can be done at any point in development (as opposed to checking out a tag which requires creating tags).

Reply:
Whoops, that was not intended. I missed it while I was reviewing the code used during testing. I thought using master could actually be safe enough but I see no harm in setting the commit hash instead.

Reply:
Now that you made me think about it, I do not think it would be a bad idea to use tags in the PKGBUILD. Of course now can't be used since the last tag does not have PKGBUILD support but could be used since the next one. What do you think?

Reply:
As I mentioned earlier, I don't think tags will work well for development builds. With tags, one would have to create tags just to try a build, or modify the PKGBUILD by hand to specify a commit.

This could be useful to allow users to test some new changes before making a release, for example.

Reply:
You are right, here is better to use the current commit. I may use tags and download the tarball instead of cloning the repo only in the PKGBUILD that I will upload to AUR, since that will use specific releases.

About the line to be fixed, I think it would be better if it is you who fix the python code to set the checkout command to the current hash since I am relativerly ignorant of python3 and I don't want to mess around.

Reply:
> I may use tags and download the tarball instead of cloning the repo only in the PKGBUILD that I will upload to AUR, since that will use specific releases.

You brought up a good point about archives; I almost forgot about them. GitHub allows you to download an archive of the tree at any commit via `https://github.com/Eloston/ungoogled-chromium/archive/${commit}.tar.gz` (another example is in the macOS base bundle extra deps). That'll save some bandwidth and space.

However, I don't quite understand your rationale for preferring tags over commits specifically for the AUR. Why go through more effort when one process can generate PKGBUILDs for both development and release builds?

> I think it would be better if it is you who fix the python code

Can do. I'll start working on it after we clarify the above issue and the new review regarding `out/Release`.

Reply:
> However, I don't quite understand your rationale for preferring tags over commits specifically for the AUR. Why go through more effort when one process can generate PKGBUILDs for both development and release builds?

In fact I don't. Perhaps I did't explained it well, speaking about tags I ment the release archives actually. I always ment to use source tarballs that GitHub generates in the PKGBUILD for the release build, and cloning git repo in the one for the developent build. This way I could also verify the hash of the downloaded archive and the "release PKGBUILD" would be more consistent and clean in my opinion.

Reply:
Ahh, I see. In that case, I will by default use the config bundle's version to determine the tag to download. I will also add a CLI option to enable use of the current commit.